wikiclue/src/routes/leaderboard/+page.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+    import HeaderBar from "../../components/headerBar.svelte";
+</script>
+
+<HeaderBar />
+<div class="leaderboards">
+    <h1>Leaderboards</h1>
+    <div class="leaderboard-container">
+        <div class="leader-container">
+            <h2>Levels</h2>
+            <p class="leader-description">Highest Level your friends have reached</p>
+            <div class ="users-container">
+                <p class="leader-name">LitLover911 <span class ="leader-score">Hard 77</span></p>
+                <p class="leader-name">Chris_Morroco <span class ="leader-score"> Hard 1</span></p>
+                <p class="leader-name">Bonnie <span class ="leader-score">Medium 211</span></p>
+                <p class="leader-name">Clyde <span class ="leader-score">Easy 5</span></p>
+            </div>
+        </div>
+        <div class="leader-container">
+            <h2>The Daily</h2>
+            <p class="leader-description">Amount of guess it took your friends to solve The Daily</p>
+            <div class ="users-container">
+                <p class="leader-name">Chris_Morroco <span class ="leader-score"> 2</span></p>
+                <p class="leader-name">Clyde <span class ="leader-score">3</span></p>
+                <p class="leader-name">Bonnie <span class ="leader-score">7</span></p>
```

Comment:
Could we make the vertical spacing between each row of name and score a bit less for all three leaderboards? And could we make the font size a bit smaller? Therefore we can fit more users on each leaderboard and also accommodate longer usernames.